tests/accelerators/test_ipu.py

```diff
@@ -343,7 +343,7 @@ def test_autoreport(tmpdir):
     )
     trainer.fit(model)
     assert os.path.exists(autoreport_path)
-    assert os.path.isfile(autoreport_path + "profile.pop")
+    assert os.path.isfile(autoreport_path + "training/profile.pop")
```

Comment:
Could you mention to the reviewers why this change is required?

Reply:
And would it affect other parts of the codebase?

Reply:
There was a change in 2.2 to allow people to pass a modelName, which is used to place profiles in separate directories. If it's not specified, it defaults to `inference` / `training`.

https://docs.graphcore.ai/projects/poptorch-user-guide/en/latest/reference.html#poptorch.Options.modelName